cobbler/actions/buildiso/__init__.py

```diff
@@ -231,7 +322,74 @@ def calculate_grub_name(self, distro) -> str:
             f' Found: "{str(matches.values())}"'
         )
 
-    def __prepare_buildisodir(self, buildisodir: str = "") -> str:
+    def _write_isolinux_cfg(
+        self, cfg_parts: List[str], output_dir: pathlib.Path
+    ) -> None:
+        """Write isolinux.cfg.
+
+        :param cfg_parts: List of str that is written to the config, joined by newlines.
+        :param output_dir: pathlib.Path that the isolinux.cfg file is written into.
+        """
+        output_file = output_dir / "isolinux.cfg"
+        self.logger.info("Writing %s", output_file)
+        with open(output_file, "w") as f:
+            f.write("\n".join(cfg_parts))
+
+    def _write_grub_cfg(self, cfg_parts: List[str], output_dir: pathlib.Path) -> None:
+        """Write grub.cfg.
+
+        :param cfg_parts: List of str that is written to the config, joined by newlines.
+        :param output_dir: pathlib.Path that the grub.cfg file is written into.
+        """
+        output_file = output_dir / "grub.cfg"
+        self.logger.info("Writing %s", output_file)
+        with open(output_file, "w") as f:
+            f.write("\n".join(cfg_parts))
+
+    def _create_esp_image_file(self, tmpdir: str) -> str:
+        esp = pathlib.Path(tmpdir) / "efi"
```

Comment:
I would like somewhere in docstring or comment explained that esp = EFI system portion. That is not straightforward IMHO (I had to google it).

Reply:
sure, I haven't added all the docstrings yet.